StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdatePaymentMethodViewController.swift

```diff
@@ -95,6 +95,9 @@ final class UpdatePaymentMethodViewController: UIViewController {
             button.contentEdgeInsets = UIEdgeInsets(top: 10, left: 16, bottom: 10, right: 16)
         }
         button.setTitleColor(viewModel.appearance.colors.danger, for: .normal)
+        button.setTitleColor(viewModel.appearance.colors.danger.disabledColor, for: .highlighted)
+        button.addTarget(self, action: #selector(buttonTouchDown(_:)), for: .touchDown)
+        button.addTarget(self, action: #selector(buttonTouchUp(_:)), for: [.touchUpInside, .touchUpOutside])
```

Comment:
Why do we need `touchUpOutside`?

Reply:
If you hold down and press the button but then drag your finger and lift it outside of the bounds of the button, without touchUpOutside, it won't go back to its original color